### DISCUSSION TITLE
题主用的neovim版本是8吗

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [nshen/learn-neovim-lua#117] [Q&A]
自从更新neovim到8后 telescope 的current buffer find就不行了
报错
![image](https://user-images.githubusercontent.com/43649186/189810565-01b92c9b-3dc9-437a-984e-f7c551463045.png)

**Comment by nshen** [comment]
7是稳定版，8可能经常更新telescope还没跟上吧?

我是用的7，试了一下 current_buffer_fuzzy_find 可用。 我还没用过这个命令哈，好像不错，我土老帽都用 `/` 搜索

**Reply by HUAHUAI23** [reply]
我看到8对那个icon支持挺好的 就升级了8，我也要学习/搜索了 哈哈哈
